app/src/main/java/io/gnosis/safe/ui/settings/owner/list/OwnerListAdapter.kt

```diff
@@ -107,12 +109,21 @@ class LocalOwnerViewHolder(private val viewBinding: ItemOwnerLocalBinding) : Bas
     }
 }
 
+private fun OwnerViewData.LocalOwner.getImageResForKeyType(): Int =
```

Comment:
move it to OwnerViewData class? Also there is no need to have a single LocalOwner subclass of the OwnerViewData so the can be combined (leave only OwnerViewData)

Reply:
Done in 23fa5d6c135a5d7b5fddf596a4a5cf01894fcf4e